src/io/syncsocket.c

```diff
@@ -434,19 +434,32 @@ MVMint64 socket_getport(MVMThreadContext *tc, MVMOSHandle *h) {
     return port;
 }
 
+static MVMint64 socket_is_tty(MVMThreadContext *tc, MVMOSHandle *h) {
+    MVMIOSyncSocketData *data = (MVMIOSyncSocketData *)h->body.data;
+    return isatty(data->handle);
```

Comment:
Would a socket ever be a TTY? I think this will break the Windows build; it'll need [something like this](https://github.com/MoarVM/MoarVM/blob/master/src/io/syncfile.c#L23) doing.

Reply:
I'm not sure if it's actually possible or not, but I added it just to be safe